sympy/combinatorics/tensor_can.py

```diff
@@ -763,12 +763,12 @@ def canonicalize(g, dummies, msym, *v):
     """
     from sympy.combinatorics.testutil import canonicalize_naive
     if not isinstance(msym, list):
-        if not msym in [0, 1, None]:
+        if not msym in (0, 1, None):
```

Comment:
Not really important but I think `x not in y` is generally better for clarity then `not x in y` and also it's very slightly faster.